lib/broccoli/glimmer-app.ts

```diff
@@ -50,16 +59,21 @@ const DEFAULT_TS_OPTIONS = {
 
 export interface OutputPaths {
   app: {
-    html: string
+    html: string,
+    js: string
   }
 }
-
 export interface EmberCLIDefaults {
   project: Project
 }
 
 export interface Options {
```

Comment:
https://github.com/glimmerjs/glimmer-application-pipeline/pull/65 addresses this comment